tests/test_folder_structure.py

```diff
@@ -0,0 +1,127 @@
+from unittest import TestCase
+from vcr import VCR
+import os
+import shutil
+from click.testing import CliRunner
+from icloudpd.base import main
+from tests.helpers.print_result_exception import print_result_exception
+
+vcr = VCR(decode_compressed_response=True)
+
+class FolderStructureTestCase(TestCase):
+
+    # This is basically a copy of the listing_recent_photos test #
+    def test_default_folder_structure(self):
+        ### Tests if the default directory structure is constructed correctly ###
+        if os.path.exists("tests/fixtures/Photos"):
+            shutil.rmtree("tests/fixtures/Photos")
+        os.makedirs("tests/fixtures/Photos")
+
+        # Note - This test uses the same cassette as test_download_photos.py
+        with vcr.use_cassette("tests/vcr_cassettes/listing_photos.yml"):
+            # Pass fixed client ID via environment variable
+            os.environ["CLIENT_ID"] = "DE309E26-942E-11E8-92F5-14109FE0B321"
+            runner = CliRunner()
+            result = runner.invoke(
+                main,
+                [
+                    "--username",
+                    "jdoe@gmail.com",
+                    "--password",
+                    "password1",
+                    "--recent",
+                    "5",
+                    "--only-print-filenames",
+                    "--no-progress-bar",
+                    "-d",
+                    "tests/fixtures/Photos",
+                ],
+            )
+            print_result_exception(result)
+            filenames = result.output.splitlines()
+
+            self.assertEqual(len(filenames), 8)
+            self.assertEqual(
+                "tests/fixtures/Photos/2018/07/31/IMG_7409.JPG", filenames[0]
```

Comment:
Please use normpath to allow tests to be executed on other platforms, e.g. Windows. See other tests as an example.